assets/airports/lszh.json

```diff
@@ -0,0 +1,1812 @@
+            "exitPoints": {
+                "WIL": [["WIL", "A70+"]]
+            },
+            "draw": [
+                ["BREGO", "ZH555", "ZH551", "WIL*"]
+            ]
+        },
+        "WIL2G": {
+            "icao": "WIL2G",
+            "name": "Willisau Two Golf",
+            "rwy": {
+                "LSZH34": ["^_RWY34KLO02DME", ["^_RWY34KLO04DME", "A35+"], "_TRA190011", ["BREGO", "A50+"]]
+            },
+            "body": [["ZH555", "A60+"], ["ZH551", "A70+"]],
+            "exitPoints": {
+                "WIL": [["WIL", "A70+"]]
+            },
+            "draw": [
+                ["BREGO", "ZH555", "ZH551", "WIL*"]
+            ]
+        },
+        "ZUE1D": {
+            "icao": "ZUE1D",
+            "name": "Zurich East One Delta",
+            "rwy": {
+                "LSZH10": [["^_RWY10KLO02.1DME", "S210-"], ["_ZUE232007", "S210-"]]
+            },
+            "body": [],
+            "exitPoints": {
+                "ZUE": [["^ZUE", "A60+"]]
+            },
+            "draw": [
+                ["_ZUE232007", "ZUE*"]
+            ]
+        },
+        "ZUE2R": {
+            "icao": "ZUE2R",
+            "name": "Zurich East Two Romeo",
+            "rwy": {
+                "LSZH16": [["^_RWY16KLO01DME", "S210-"], ["_ZUE232007", "S210-"]]
+            },
+            "body": [],
+            "exitPoints": {
+                "ZUE": [["^ZUE", "A60+"]]
+            },
+            "draw": [
+                ["_ZUE232007", "ZUE*"]
+            ]
+        },
+        "ZUE2V": {
+            "icao": "ZUE2V",
+            "name": "Zurich East Two Victor",
+            "rwy": {
+                "LSZH28": [["^_RWY28KLO03DME", "S210-"], ["_KLO253005", "S210-"], ["^ZH552", "S210-"], "_KLO232005", "KLO"]
+            },
+            "body": [],
+            "exitPoints": {
+                "ZUE": [["^ZUE", "A60+"]]
+            },
+            "draw": [
+                ["_ZUE232007", "ZUE*"]
+            ]
+        },
+        "ZUE3L": {
+            "icao": "ZUE3L",
+            "name": "Zurich East Three Lima",
+            "rwy": {
+                "LSZH32": ["^_RWY34KLO02DME", ["^_RWY34KLO04DME", "A35+|S210-"], "_TRA190011", ["^_TRA190012.5", "A45+"], "_KLO269005.5", ["KLO", "A50+"]]
+            },
+            "body": [],
+            "exitPoints": {
+                "ZUE": [["^ZUE", "A60+"]]
+            },
+            "draw": [
+                ["_ZUE232007", "ZUE*"]
+            ]
+        },
+        "ZUE3F": {
+            "icao": "ZUE3F",
+            "name": "Zurich East Three Foxtrott",
+            "rwy": {
+                "LSZH34": ["^_RWY34KLO02DME", ["^_RWY34KLO04DME", "A35+|S210-"], "_TRA190011", ["^_TRA190012.5", "A45+"], "_KLO269005.5", ["KLO", "A50+"]]
+            },
+            "body": [],
+            "exitPoints": {
+                "ZUE": [["^ZUE", "A60+"]]
+            },
+            "draw": [
+                ["_ZUE232007", "ZUE*"]
+            ]
+        }
+    },
+    "stars": {

```

Comment:
```
AMIKI28
AMIKI34
GIPOL28
GIPOL34
RILAX28
RILAX34
```
These transitions can be useful, I would advise putting them in unless you have objections